catalog/video/video-admin-plugin/src/main/webapp/js/model/StreamMonitor.js

```diff
@@ -80,7 +80,7 @@ define(['backbone',
                     $.each(configurations, function(index, value) {
                         var url = value.monitoredAddress;
                         var maxDuration = value.elapsedTimeRolloverCondition / (60 * 1000);
-                        var maxSize = value.byteCountRolloverCondition / (1000 * 1000);
+                        var maxSize = value.byteCountRolloverCondition;
```

Comment:
✏️ If these values are in MB, should we name them `maxMegabyteSize` and `megabyteCountRolloverCondition`?